### DISCUSSION TITLE
Compile failure & solution

### DISCUSSION THREAD

**Discussion by PetitPipi** [Anura3D/Anura3D_OpenSource#159] [Q&A] [answered]
Hi, 
The compiling failed with the up to date code (2023-11-17).  Then I looked into the log, found that line 55 in WriteOutPut_GiD.for might be the problem.  So I comment out this line, and it works.  Hope this helps. 

The line is:
#pragma comment(lib, "gidpost.lib")

Regards,
P

**Comment by murphy-john** [comment, marked as answer]
Yes, I and others have noticed this issue. The fix is to comment this line out. You can comment the line out by adding an exclamation point "!" to the beginning of that line. I was able to compile the code and run models after doing so.
